spec/unit/xml_params_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+describe XmlParams do
+  source_filepath = './tmp/86476809/(NS)-CHAUZIMU-MWA-CHILENGEDWE.doc'
+  converted_filepath = './tmp/331411115.docx'
+  fonts_path = './assets/fonts'
+  format = :docx
+  xml = described_class.new(fonts_path: fonts_path, tmp_path: StaticData::TMP_DIR)
+  xml_template = File.open('./spec/unit/template.xml') { |f| Nokogiri::XML(f) }
+  created_xml = File.open(xml.create_xml(source_filepath, converted_filepath, format)) { |f| Nokogiri::XML(f) }
```

Comment:
`parsed_result` is better name, because is not `created` but also `parsed` via Nokogiri